packages/clay-management-toolbar/src/ClayManagementToolbar.soy

```diff
@@ -367,6 +373,19 @@
 					{/call}
 				</li>
 
+				{if $infoButton}
```

Comment:
For consistency with the rest of the params I think this should be named `showInfoButton`